go.mod

```diff
@@ -17,10 +17,12 @@ require (
 	github.com/gorilla/securecookie v1.1.1
 	github.com/gorilla/sessions v1.2.1
 	github.com/iancoleman/strcase v0.2.0
+	github.com/jackc/pgconn v1.10.1 // indirect
```

Comment:
what does the indirect mean?

Reply:
It's a dependency that is not directly used by our code, but is required by another dependency (in this case `gorm.io/gorm`, which is our database connection). 
It's needed because this package exposes the errors that are returned by the database, and we need to check the database error and map it to an api error, for example `ErrUniqueConstraint` -> `HTTP 409 Conflict`

Reply:
It's also just used in our tests